js/main.js

```diff
@@ -153,3 +177,21 @@ document.addEventListener('DOMContentLoaded', function (event) {
     viewSwapDataViews(data.view);
   }
 });
+
+// eventlistener 'click' for document
+document.addEventListener('click', function (event) {
+  var links = document.getElementsByTagName('a');
+
+  for (var i = 0; i < links.length; i++) {
+    if (event.target.tagName !== links[i].tagName) {
+      return;
+    } else if (event.target.tagName === links[i].tagName) {
```

Comment:
You will be able to remove this else if. This could just be an else, but since you are returning in the conditional block above, the conditionals you have in this conditional block can just fall below the conditional you are revising to check if the tag clicked was an anchor tag 👍